src/utils/index.js

```diff
@@ -19,6 +19,149 @@ function logInfo(message) {
   console.log(blue(message));
 }
 
+function logHelp(command) {
+  if (!command) {
+    console.log(`
+Usage:
+  blazepack   Start dev server on existing project
+  blazepack   [options]  [command]
+
+Options:
+  --version   Output the current version
+  --port <p>  Specify port <p> for dev server
+
+Commands:
+  create          Create project from Template
+  start           Create project from Template & Start the dev server
+  clone           clone an existing codesandox
+  export          Export to codesandbox.io
+  install         Install dependency
+  add             Install dependency (alias for install)
+  uninstall       Remove dependency
+  remove          Remove dependency (alias for uninstall)
+  help [command]  show help for a particular command
+`);
+  } else {
+    switch (command) {
+      case 'create':
+        console.log(`
+Usage: blazepack create <name> [--template=<template>]
+        
+Create project with name <name> from Template <template>.
+If the --template option is not specified you will get the
+list of available templates to select from. 
+
+eg: blazepack create my-cra --template=react
+
+List of available templates are:
+  1. static
```

Comment:
please remove the numbers from here

Reply:
also fetch the list from a constant which will be shared, so that when we add a new template tmrw there would not be a need to update it in the help command

Reply:
should i create this list of template names in an array called `TEMPLATES` in the `constants.js` file.?

Reply:
Yes @dhruwlalan and also share the constant with other commands like clone and create

Reply:
how would i use it in the clone command?

Reply:
https://github.com/ameerthehacker/blazepack/blob/master/bin/blazepack.js#L42
in this validation and in some places we can reuse the same constant